jobs/scheduled/enqueue_reminders.rb

```diff
@@ -17,9 +17,18 @@ def skip_enqueue?
 
     def allowed_group_ids
       allowed_groups = SiteSetting.assign_allowed_on_groups.split('|')
+      return [] if allowed_groups.empty?
+
       Group.where(name: allowed_groups).pluck(:id).join(',')
```

Comment:
I would expect a method called `allowed_group_ids` to return a collection of ids, not a string. I think it would be nicer if you returned the `pluck` here and had the `membership_condition` method call `join`